ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-fcos-presubmits.yaml

```diff
@@ -87,7 +87,8 @@ presubmits:
         - name: JOB_NAME_SAFE
           value: e2e-aws
         - name: TEST_COMMAND
-          value: TEST_SUITE=openshift/conformance/parallel run-tests
+          value: |
+            TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
What does the pipe here do?

Reply:
it's not supposed to change anything functionally, just to trigger an e2e-aws rehearsal here, see Clayton's comment above.

Reply:
Ah ok, just wanted to make sure that's what it was.

Reply:
(also, this is just my naive assumpion; I've seen multi-line commands written this way (with the pipe) elsewhere). It did trigger the e2e test this time, so let's see whether or not it works! =)

Reply:
`|` just makes it multiline.

Reply:
will it still work with just one line?

Reply:
(otherwise I'll pull `run-tests` down in a second line)